pkg/server/router/webhook.go

```diff
@@ -35,12 +35,11 @@ type CreateWebhookRequest struct {
 	Noun webhook.Noun `json:"noun" validate:"required"`
 	// The verb for the new webhook.eg: Create
 	Verb webhook.Verb `json:"verb" validate:"required"`
-	// The URLS to post the output of this request to Noun.Verb action to.
-	URLS []string `json:"urls" validate:"required"`
+	// The URL to post the output of this request to Noun.Verb action to.
+	URL string `json:"url" validate:"required"`
```

Comment:
It felt strange having a CreateWebhook route being able to create multiple webhooks urls (almost like we would make a new route called CreateWebhooks)

Also a bit more complexity on deduping urls.
 
But I can add it in if you think it would be beneficial and used like this

Reply:
I think either way is fine. Was just curious. I agree that this seems more natural.